src/topoflow_glacier/bmi/bmi_topoflow_glacier.py

```diff
@@ -1218,6 +1396,53 @@ def update_net_longwave_radiation(self):
                [K].  Recall that absolute zero occurs at
                0 [deg_K] or -273.15 [deg_C].
 
+        ----------------------------------------------------------------
+        First, e_air is computed as:
+          e_air = RH * 0.611 * exp[(17.3 * T_air) / (T_air + 237.3)]
+        Then, em_air is computed as:
+          em_air = (1 - F) * 1.72 * [e_air / (T_air + 273.15)]^(1/7) *
+                    (1 + 0.22 * C^2) + F
+        ----------------------------------------------------------------
+        Compute Qn_LW for this time
+        --------------------------------
+        """
+        #logger.info("update_net_longwave_radiation")
+        T_surf_K = self.T_surf + self.C_to_K
+        T_air_K  = self.T_air  + self.C_to_K
+
+        # Fast path: if LWDOWN is provided in forcing, use it for LW_in.
+        try:
+            LW_in_forced = np.asarray(self.LW_in, dtype="float64")
+        except Exception:
```

Comment:
this exception probably gives you pre-commit hooks errors, because it is not especified. unless pre-commit hooks has been silenced.